src/webcore/try_from.rs

```diff
@@ -27,3 +27,15 @@ impl< T, U > TryInto< U > for T where U: TryFrom< T > {
         U::try_from( self )
     }
 }
+
+impl< T, U > TryFrom< Option<T> > for Option<U>
```

Comment:
Please get rid of this impl. In the future I'd like to get rid of our `TryFrom` and just use one from the stdlib, and this impl will make that harder.